components/temporarycomponents/YoutubeLinkBanner.tsx

```diff
@@ -0,0 +1,90 @@
+import { useTranslation } from 'next-i18next';
+import { getCountryVariant } from 'utils/locales';
+import YoutubeLinkIcon from 'public/icons/youtube-link.svg';
+import Image from 'next/image';
+import { ReactNode } from 'react';
+import Lekarna_UK_mobile from 'public/youtubebanner/lekarna-uk--mobile.png';
+import Lekarna_CZ_mobile from 'public/youtubebanner/lekarna-cz--mobile.png';
+import Lekarna_UK_desktop from 'public/youtubebanner/lekarna-uk--desktop.png';
+import Lekarna_CZ_desktop from 'public/youtubebanner/lekarna-cz--desktop.png';
+import Person_UK from 'public/youtubebanner/person-uk.png';
+import Person_CZ from 'public/youtubebanner/person-cz.png';
+
+const IMAGES: Record<string, ReactNode> = {
+  'lekarna-uk-mobile': <Image src={Lekarna_UK_mobile} alt="Ikona se slovem aптека" />,
+  'lekarna-cz-mobile': <Image src={Lekarna_CZ_mobile} alt="Ikona se slovem lekárna" />,
+  'lekarna-uk-desktop': <Image src={Lekarna_UK_desktop} alt="Ikona se slovem aптека" />,
+  'lekarna-cz-desktop': <Image src={Lekarna_CZ_desktop} alt="Ikona se slovem lekárna" />,
+};
+
+const YTVIDEOCODECZ: Array<string> = [
+  'yf40iMpWM3c', //1 - Lékárna
+  'v0MDWXL281o', //2 - Na masáži
+  'gN_rr3CevqQ', //3 - Muchomůrka zelená
+  'vx6f8F8mvfs', //4 - Ovoce
+  'JEZpbOqFak0', //5 - U doktora
+  '4JdK3Hc5gCU', //6 - Na poště
+  'mgpjnYsTAnA', //7 - Na nákupu
+  'IJ6RH8jGPII', //8 - V zoo
+  'BujRB73XbaI', //9 - U krejčího
+];
+
+const YTVIDEOCODEUK: Array<string> = [
+  'pLMWJcbV0eA', //10 - Аптека
+  '0lZYqftgdc8', //11 - На масажі
+  '9THEohGeCZ4', //12 - Поганка
+  'EU6g6VnWSW8', //13 - Фрукти
+  'XPOQd_EVrh0', //14 - У лікаря
+  'hiLHhYXBo2s', //15 - У кравця
+  'S_eRJBkp_Zc', //16 - На пошті
+  'AjK635bgEZo', //17 - На шопінг
+  'KMz8hsfnC3Q', //18 - У зоопарку
+];
+
+export const YoutubeLinkBanner = () => {
+  const { t, i18n } = useTranslation();
+  const isUk = i18n.language === 'uk';
```

Comment:
You can use our `useLanguage` hook to get a `currentLanguage` in a more narrowly typed way :)